### ISSUE TITLE
SentenceSplittingMode.All ignored 

### ISSUE THREAD

**Issue by nicStuff** [DeepLcom/deepl-java#3]
As per-title, the only place reading the `SentenceSplittingMode` ignores the `All` value, which is useful when setting the tag handing to html.

https://github.com/DeepLcom/deepl-java/blob/63536dc9d0c21472c8cf20fa1acbd7c8bad1d608/deepl-java/src/main/java/com/deepl/api/Translator.java#L769

Maybe it's because the default has changed since november 2022 as stated in https://www.deepl.com/docs-api/translate-text/translate-text/:

`The use of nonewlines as the default value for text translations where tag_handling=html is new behavior that was implemented in November 2022, when HTML handling was moved out of beta.`

I have tried to request a translation through a `curl` request for the same input and the result is just fine.

**Comment by nicStuff** [comment]
Here's a patch for fixing the issue as I did locally

```
--- a/deepl-java/src/main/java/com/deepl/api/Translator.java
+++ b/deepl-java/src/main/java/com/deepl/api/Translator.java
@@ -765,8 +765,7 @@ public class Translator {
 
     if (options != null) {
       // Note: formality and glossaryId are added above
-      if (options.getSentenceSplittingMode() != null
-          && options.getSentenceSplittingMode() != SentenceSplittingMode.All) {
+      if (options.getSentenceSplittingMode() != null) {
         switch (options.getSentenceSplittingMode()) {
           case Off:
             params.add(new KeyValuePair<>("split_sentences", "0"));
@@ -774,6 +773,9 @@ public class Translator {
           case NoNewlines:
             params.add(new KeyValuePair<>("split_sentences", "nonewlines"));
             break;
+          case All:
+            params.add(new KeyValuePair<>("split_sentences", "1"));
+            break;
           default:
             break;
         }
```

**Comment by daniel-jones-deepl** [comment]
Hi @nicStuff, thanks for reporting this bug and suggesting the fix. I'll push a new version soon.

**Comment by daniel-jones-deepl** [comment]
This is fixed in [v1.0.1](https://github.com/DeepLcom/deepl-java/releases/tag/v1.0.1).
